calblueprint/ks-web
# Andi/farmediting2

[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

[//]: # '####### YOUR ANSWER BELOW ###########'
- Replace placeholders of FieldInput with the farm data that matches the record ID. 
- Editing should validate email address, mailing address, and phone numbers and display an error state (enabled via MUI) if not valid (see utils.js for field validation functions) 

### How to Review
- Check that grey placeholders match the Airtable data
- Check that incorrect values have errors. 
-

[//]: # 'The order in which to review files and 
what to expect when testing locally'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # '####### YOUR ANSWER BELOW ###########'
I'm honestly still confused about Autocomplete. If it's as simple as setting something to true, maybe someone could look at my code and give that a go? 

### Screenshots

[//]: # '#### YOUR SCREENSHOTS BELOW ########'
![image](https://user-images.githubusercontent.com/43193441/114962778-33322b00-9e20-11eb-85dd-50755c937fb3.png)

### Design Status

[//]: # 'If this is a frontend PR, what is the expected 
status of the UI in this PR (lo, mid, high- fi?'
[//]: # '####### LOFI/MIDFI/HIFI ###########'
[//]: # '############## END ##################'

CC: @phoebeli23
